### ISSUE TITLE
TypeError: Cannot read property 'uc_base' of undefined

### ISSUE THREAD

**Issue by dbrgn** [Ranks/emojione#494]
Steps to reproduce:

```javascript
> emojione.unicodeToImage('#')
http://[::1]:9966/libs/emojione/emojione.js:333 Uncaught TypeError: Cannot read property 'uc_base' of undefined
    at http://[::1]:9966/libs/emojione/emojione.js:333
    at String.replace (<anonymous>)
    at Object.ns.unicodeToImage (http://[::1]:9966/libs/emojione/emojione.js?v=:315)
    at <anonymous>:1:10
```

The problem is probably in the `if` statements on lines 316-327, none of them matches which leaves `fname` undefined.

**Comment by caseyahenson** [comment]
This is fixed in the 3.0.3 library patch.

**Comment by luichanfe** [comment]
In my case, solved setting **greedyMatch** property to true
